### ISSUE TITLE
loss function did not converge

### ISSUE THREAD

**Issue by CasparZhao** [Bob-cheng/MDE_Attack#2]
When I was training, I found that the loss function did not converge. Is this normal?

**Comment by zhanz5** [comment]
Did you solve the problem? I had the same problem.
